src/components/core/InteractiveLink/InteractiveLink.tsx

```diff
@@ -81,17 +95,59 @@ export default function InteractiveLink({
   return null;
 }
 
-export const StyledAnchor = styled.a<{ disabled?: boolean }>`
-  color: ${({ disabled }) => (disabled ? colors.porcelain : colors.shadow)};
+export function InteractiveLinkNeedsVerification({
+  inheritLinkStyling,
+  href,
+  children,
+}: {
+  inheritLinkStyling: boolean;
+  href: string;
+  children: ReactNode;
+}) {
+  const { showModal } = useModalActions();
+  const handleClick = useCallback(
+    (href) => {
+      // track('What to track here?');
+
+      // FIXME: This conflicts with viewing the NFT preview page, because that is a modal.
+      // So running `showModal` will close that preview modal, and then open the verify navigation modal.
+      showModal({
+        content: <NavigateConfirmation href={href} />,
+        isFullPage: false,
+        headerText: `Are you sure you want to navigate to ${href}?`,
+      });
+    },
+    [showModal]
+  );
+  return (
+    <InteractiveLink
+      inheritLinkStyling={inheritLinkStyling}
+      onClick={() => {
+        handleClick(href);
+      }}
+    >
+      {children}
+    </InteractiveLink>
+  );
+}
+
+export const StyledAnchor = styled.a<{ disabled?: boolean; inheritStyles?: boolean }>`
   text-decoration: underline;
-  font-family: ${BODY_FONT_FAMILY};
-  font-size: 14px;
-  line-height: 18px;
+  color: ${({ disabled }) => (disabled ? colors.porcelain : colors.shadow)};
   transition: color ${transitions.cubic};
   cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'pointer')};
 
   &:hover {
     text-decoration: none;
     color: ${colors.offBlack};
   }
+
+  ${({ inheritStyles }) =>
+    inheritStyles
+      ? ''
+      : css`
+          font-family: ${BODY_FONT_FAMILY};
+          font-size: 14px;
+          line-height: 18px;
+        `}
```

Comment:
interesting, do we need to wrap in `css` here?

we might also be able to do something like:
```js
font-family: ${({ inheritStyles }) => ? 'inherit' : BODY_FONT_FAMILY};
font-size: ${({ inheritStyles }) => ? 'inherit' : '14px'};
line-height: ${({ inheritStyles }) => ? 'inherit' : '18px'};
```